### PR TITLE
TELCODOCS-1977: Document tagging behaviour

### PR DESCRIPTION
**Pull request by StephenJamesSmith**
D/S Docs:[MGMT-16637] Document tagging behaviour

Jira: https://issues.redhat.com/browse/TELCODOCS-1977

Version(s): openshift-4.17

Fix version: KMMO 2.2

Link to docs preview: https://81803--ocpdocs-pr.netlify.app/openshift-enterprise/latest/hardware_enablement/kmm-kernel-module-management.html#kmm-example-cr_kernel-module-management-operator

Dev review: @ybettan 
QE review: @cdvultur

### CODE REVIEW

File: modules/kmm-example-module-cr.adoc [hunk 1]
```diff
@@ -77,7 +77,7 @@ spec:
 <3> Optional: Copies `/firmware/*` into `/var/lib/firmware/` on the node.
 <4> Optional.
 <5> At least one kernel item is required.
-<6> For each node running a kernel matching the regular expression, KMM creates a `DaemonSet` resource running the image specified in `containerImage` with `${KERNEL_FULL_VERSION}` replaced with the kernel version.
+<6> For each node running a kernel matching the regular expression, KMM checks if you have included a tag or a digest. If have not specified a tag or digest in the container image, then the validation webhook returns an error and dows not apply the module.
```
Comment (ybettan) [inline modules/kmm-example-module-cr.adoc:80]:
```suggestion
<6> For each node running a kernel matching the regular expression, KMM checks if you have included a tag or a digest. If have not specified a tag or digest in the container image, then the validation webhook returns an error and does not apply the module.
```

**Comment by ybettan** [comment]
@StephenJamesSmith I believe that @qbarrand isn't the person you need for QE review. You may want to tag someone from QE.
@cdvultur perhaps.

File: modules/kmm-example-module-cr.adoc (hunk 1 above)
Comment (StephenJamesSmith) [reply modules/kmm-example-module-cr.adoc:80]: Fixed.

**Comment by StephenJamesSmith** [comment]
@ybettan @cdvultur Please review or /lgtm.

**Review by snarayan-redhat — COMMENTED** [review]
lgtm

**Pull request merged**